nuage-tripleo-heat-templates/environments/neutron-nuage-config.yaml

```diff
@@ -0,0 +1,43 @@
+# A Heat environment file which can be used to enable a
+# a Neutron Nuage backend on the controller, configured via puppet
+resource_registry:
+  OS::TripleO::Services::NeutronDhcpAgent: OS::Heat::None
+  OS::TripleO::Services::NeutronL3Agent: OS::Heat::None
+  OS::TripleO::Services::NeutronMetadataAgent: OS::Heat::None
+  OS::TripleO::Services::NeutronOvsAgent: OS::Heat::None
+  OS::TripleO::Services::ComputeNeutronOvsAgent: OS::Heat::None
+  # Override the NeutronCorePlugin to use Nuage
+
+  OS::TripleO::Docker::NeutronMl2PluginBase: puppet/services/neutron-plugin-ml2-nuage.yaml
```

Comment:
the path should be ../puppet/services, because this file is in environments folder and need to go 1 step back